crud.lua

```diff
@@ -25,25 +25,37 @@ crud.register = registry.add
 --- CRUD operations.
 -- @section crud
 
--- @refer insert.call
+-- @refer insert.call_raw
 -- @function insert
-crud.insert = insert.call
+crud.insert = insert.call_raw
+
+-- @refer insert.call
+-- @function insert_object
+crud.insert_object = insert.call
```

Comment:
It looks a bit strange:

```lua
crud.insert = insert.call_raw
crud.insert_object = insert.call
```

It seems that default flow for `insert` module is "objects" flow, but `crud` uses "raw" flow by default.

```lua
crud.insert = insert.raw
crud.insert_object = insert.obj
```

So, `crud` module decides which flow is default, while `insert` module just introduces two different functions.

Reply:
I've change a bit naming convention here - https://github.com/tarantool/crud/pull/40

insert.tuple - for tuples
insert.object - for objects

Reply:
Naming convention was changed in previous patch. This patch only changes API and test them.